package.json

```diff
@@ -74,8 +73,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "nhsuk-frontend": "^2.1.0",
-    "unistore": "^3.4.1"
```

Comment:
I removed this dependency as part of this branch - I added it with investigation into the new forms.